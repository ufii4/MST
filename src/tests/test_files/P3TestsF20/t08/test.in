n 0
n 1
degree 0
degree 1
edge_count
mst
i 0;1;5.5
edge_count
d 0;1
edge_count
mst
clear
edge_count
exit
