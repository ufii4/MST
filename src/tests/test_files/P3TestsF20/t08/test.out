invalid argument
success
degree of 0 is 0
invalid argument
edge count is 0
mst 0.00
invalid argument
edge count is 0
invalid argument
edge count is 0
mst 0.00
success
edge count is 0
